src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -0,0 +1,162 @@
+package io.strimzi.kafka.bridge.http;
+
+import io.strimzi.kafka.bridge.ConnectionEndpoint;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Future;
+import io.vertx.core.http.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+public class HttpBridge extends AbstractVerticle {
+
+    private static final Logger log = LoggerFactory.getLogger(HttpBridge.class);
+
+    private HttpServer httpServer;
+
+    private HttpBridgeConfigProperties bridgeConfigProperties;
+
+    private Map<HttpConnection, ConnectionEndpoint> endpoints;
+
+    private boolean isReady;
+
+    @Autowired
+    private void setBridgeConfigProperties(HttpBridgeConfigProperties httpBridgeConfigProperties){
```

Comment:
use space before opening the `{` as a general role on style so applied to the other lines of code in this PR